### PR TITLE
linkedList new functions added

### PR DESCRIPTION
**Pull request by oflocon**
Changed linkedList.chpl and linkedList.good files

### CODE REVIEW

**Comment by lydia-duncan** [comment]
Great, thanks!  Since the code itself is in our release/examples hierarchy, we tend to be stricter about a couple of things: we'd like to have the indentation unified across the entire file, and it'd be good to match the overall style as much as possible (spaces between = signs, curly braces in the same spot, line feeds following ; etc.)  Would you mind cleaning that up a little?  Last thing to fix, I swear!

**Comment by lydia-duncan** [comment]
Okay, this looks good to go!  I'll merge it when the green checkmark comes up.  Thanks!

File: test/release/examples/programs/linkedList.chpl [hunk 1]
```diff
@@ -36,7 +36,61 @@ class List {
...
   }
+ 
+  
+ //  
+ // deletes the first element from the list 
+ //
+
+ proc delete_first() {
+    if head == nil then {
+      writeln("list is empty");
```
Comment (mppf) [inline test/release/examples/programs/linkedList.chpl]: This writeln (and the following two) should probably be deleted. This is an example program, but it wouldn't generally be OK to put such writeln calls in a library routine.

File: test/release/examples/programs/linkedList.chpl [hunk 2]
```diff
@@ -36,7 +36,61 @@ class List {
...
+    delete current ;
+  }
+
+ //
+ // deletes the element at position p from the list 
+ //
+ 
+ proc delete_pos(pos: int) {
+    if head==nil then {
+      writeln("list is empty");
```
Comment (mppf) [inline test/release/examples/programs/linkedList.chpl]: Probably remove this writeln.

File: test/release/examples/programs/linkedList.chpl [hunk 3]
```diff
@@ -36,7 +36,61 @@ class List {
...
+    var current=head;
+    
+
+    while(count!=pos-1 && current!=nil) {  
+     current=current.next;
+     count=count+1;
+    }
+   
+    if current.next==nil then  { 
+     writeln("No element found at position p");
```
Comment (mppf) [inline test/release/examples/programs/linkedList.chpl]: And this one.

**Pull request merged**